scripts/create-dumps.sh

```diff
@@ -1,41 +1,46 @@
 #!/usr/bin/env bash
```

Comment:
Let's start following https://github.com/metabrainz/metabrainz-howto-guides/blob/master/writing_effective_data_processing_scripts.md and have our data scripts return proper exit codes.

Reply:
@paramsingh 
Rob says: 
> a last line of `exit 0` should suffice.

Reply:
Done!